apps/src/aichat/views/modelCustomization/MultiInputCustomization.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {useState, useCallback} from 'react';
+
+import Button from '@cdo/apps/componentLibrary/button/Button';
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import {StrongText} from '@cdo/apps/componentLibrary/typography/TypographyElements';
+
+import {Visibility} from '../../types';
+
+import FieldLabel from './FieldLabel';
+import {isDisabled} from './utils';
+
+import styles from './retrieval-customization.module.scss';
+import modelCustomizationStyles from '../model-customization-workspace.module.scss';
+
+const MultiInputCustomization: React.FunctionComponent<{
+  label: string;
+  fieldId: string;
+  tooltipText: string;
+  addedItems: string[];
+  visibility: Visibility;
+  isReadOnly: boolean;
+  hideInputsWhenReadOnly: boolean;
```

Comment:
Would you mind explaining why this prop is needed?

Reply:
Yes! That's because in #60206 we updated the component to hide FieldLabel, text input and Add button when read only, which made sense in Retrieval because we only have the 1 customization in Retrieval - but in this context it would be a bit confusing imo to hide it and then just have "Added" at the bottom with no header, so I made it configurable.  I can rename!